internal/services/controller/controller_test.go

```diff
@@ -63,14 +61,127 @@ func TestMain(m *testing.M) {
 }
 
 func TestController_HappyPath(t *testing.T) {
-	scheme := runtime.NewScheme()
-	utilruntime.Must(karpenterCoreAlpha.SchemeBuilder.AddToScheme(scheme))
-	utilruntime.Must(karpenterAlpha.SchemeBuilder.AddToScheme(scheme))
-	utilruntime.Must(karpenterCore.SchemeBuilder.AddToScheme(scheme))
-	utilruntime.Must(karpenter.SchemeBuilder.AddToScheme(scheme))
-	utilruntime.Must(datadoghqv1alpha1.SchemeBuilder.AddToScheme(scheme))
-	utilruntime.Must(argorollouts.SchemeBuilder.AddToScheme(scheme))
+	tests := map[string]struct {
+		objectCount int
+	}{
+		"happy path": {
+			objectCount: 14,
+		},
+	}
```

Comment:
yeah, this is a refactor for other changes so there would be less visible changes when actual bug fix is implemented

Reply:
it does not add too much value as a separate PR when I looked into https://github.com/castai/k8s-agent/pull/168

Reply:
well if I wouldn't separate it would be more difficult to read the actual changes because there is test refactor where data is moving and additional logic to support bug fix